fields/types/gcsimage/GcsImageColumn.js

```diff
@@ -34,7 +34,7 @@ var GcsImageColumn = React.createClass({
     );
 
     const iptc = _.get(value, ['iptc'], {});
-    const { byline, caption, created_time, keywords } = iptc;
```

Comment:
I suppose we should rewrite the `iptc.keywords` below?

Reply:
Updated in https://github.com/twreporter/keystone/pull/194/commits/b424d963e085e315db4a5a5b8f70dfe485984ba2.